frontend/mulighetsrommet-veileder-flate/src/components/fakedoor/CountDown.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import moment from 'moment';
+import './CountDown.less';
+
+const CountDown = () => {
+  const dagerTilPilot = numberOfDaysUntilPilot()
+
+  return (
+    <div className="fakedoor-countdown">
+        <span>Tjenesten vil bli tilgjengelig om </span>
+      <span className={"fakedoor-countdown-days"}> {dagerTilPilot} </span>
+      {dagerTilPilot != 1 && <span>dager.</span>}
+      {dagerTilPilot == 1 && <span>dag.</span>}
```

Comment:
Her kan du skrive `<span>{dagerTilPilot > 1 ? 'dager.' : 'dag.'}</span>`

Så slipper du to linjer kode her som er nesten identisk.